pkg/vex/controller/reconciler.go

```
@@ -0,0 +1,181 @@
+
+	httpClient := r.HTTPClient
+	if httpClient == nil {
+		httpClient = &http.Client{Timeout: 2 * time.Minute}
+	}
+
+	resp, err := httpClient.Do(httpReq)
+	if err != nil {
+		r.setStatusFailed(vexSource, "HTTPFetchError", err.Error())
+		return reconcile.Result{RequeueAfter: 5 * time.Minute}, nil // Retry network errors
+	}
+	defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)